packages/webview_flutter/lib/webview_flutter.dart

```diff
@@ -109,6 +109,7 @@ class SurfaceAndroidWebView extends AndroidWebView {
           layoutDirection: TextDirection.rtl,
           creationParams: MethodChannelWebViewPlatform.creationParamsToMap(
             creationParams,
+            usesHybridComposition: true,
```

Comment:
We should test that creating a `SurfaceAndroidWebView` sets this in the method channel to ensure the Dart side of this stays working.

Reply:
I ended up adding an e2e test. :)

Reply:
We still really need a unit test harness for this package, but I'll take it :)